src/test/java/lambda/part2/exercise/Exercise2.java

```diff
@@ -0,0 +1,98 @@
+package lambda.part2.exercise;
+
+import lambda.data.Person;
+import org.junit.jupiter.api.Test;
+
+import java.util.function.Predicate;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+@SuppressWarnings({"ConstantConditions", "unused"})
+class Exercise2 {
+
+    @Test
+    void personHasNotEmptyLastNameAndFirstName() {
+        // TODO предикат Person -> boolean, проверяющий что имя и фамилия человека не пусты
+        Predicate<Person> validate = (p1) -> !(p1.getFirstName().isEmpty() || p1.getLastName().isEmpty());
```

Comment:
Why not use &&, it would be easier to understand for others, and u can get NullPointerException if not check firstname and lastname for null